render/detailed_node.go

```diff
@@ -128,22 +123,29 @@ func OriginTable(r report.Report, originID string) (Table, bool) {
 	return Table{}, false
 }
 
-func endpointOriginTable(nmd report.NodeMetadata) (Table, bool) {
+func connectionDetailsRows(endpointTopology report.Topology, originID string, nmd report.NodeMetadata) []Row {
 	rows := []Row{}
-	for _, tuple := range []struct{ key, human string }{
-		{"addr", "Endpoint"},
-		{"port", "Port"},
-	} {
-		if val, ok := nmd[tuple.key]; ok {
-			rows = append(rows, Row{Key: tuple.human, ValueMajor: val, ValueMinor: ""})
+	local := fmt.Sprintf("%s:%s", nmd["addr"], nmd["port"])
```

Comment:
In keeping with the tradition of const-izing the metadata keys — is this an opportunity to do the same here?